exopy_hqc_legacy/tasks/tasks/instr/transfer_awg_file_task.py

```diff
@@ -0,0 +1,484 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is probably similar enough to the other existing task to factor some common code and limit the duplication.